static/scripts/rewards/render-transaction/fetch-permits.ts

```diff
@@ -59,6 +92,10 @@ export async function readClaimDataFromUrl(app: AppState) {
   await renderTransaction();
 }
 
+async function readClaimDataFromUrl(): Promise<PermitReward[]> {
+  return decodeClaimData(base64encodedTxData ?? "");
+}
```

Comment:
Now the whole functions `readClaimDataFromUrl` look redundant. Can we just  use `decodeClaimData` and rename it to `decodeClaimDataFromUrl`